lib/models/issueish.js

```diff
@@ -16,13 +19,17 @@ export default class Issueish {
     this.createdAt = moment(data.createdAt, moment.ISO_8601);
     this.headRefName = data.headRefName;
     this.headRepositoryID = data.repository.id;
-    this.statusContexts = data.commits.nodes.reduce((acc, node) => {
-      const status = node.commit.status;
-      if (status !== null) {
-        acc.push(...status.contexts);
-      }
-      return acc;
-    }, []);
+    this.latestCommit = null;
+    this.statusContexts = [];
+    this.checkRuns = [];
+
+    if (data.commits.nodes.length === 1) {
```

Comment:
Good catch :+1:

This _shouldn't_ make a difference in practice, because the Issueish queries we make limit their `commits` connection like so:

```graphql
commits(last: 1) {
  nodes {
    # ...
  }
}
```

But I'm all for making code less brittle for the future.